JenkinsfileRT

```diff
@@ -0,0 +1,22 @@
+// Obtain files from source control system.
+if (utils.scm_checkout()) return
+
+// Define each build configuration, copying and overriding values as necessary.
+bc0 = new BuildConfig()
+bc0.nodetype = "linux-stable"
+bc0.name = "debug"
+bc0.build_cmds = ["conda env update --file=environment.yml",
+	       	  "with_env -n jwql python setup.py install"]
+bc0.test_cmds = ["cp /grp/jwst/ins/jwql/config.json jwql/utils/",
+	         "with_env -n jwql pytest -s --junitxml=result.xml"]
+bc0.failedUnstableThresh = 1
+bc0.failedFailureThresh = 1
+
+
+
+// bc1 = utils.copy(bc0)
+// bc1.build_cmds[0] = "conda install -q -y python=3.5"
```

Comment:
@SaOgaz We have been developing the project with python 3.6, should we use that here?